test_connect_delay: add missing newline at end of file

Signed-off-by: Thomas De Schampheleire <thomas.de.schampheleire@gmail.com>
Backported-from: https://github.com/zeromq/zeromq3-x/commit/143ae519b4a325bb68d29e6f7e00cb98a440dd12

diff --git a/tests/test_connect_delay.cpp b/tests/test_connect_delay.cpp
--- a/tests/test_connect_delay.cpp
+++ b/tests/test_connect_delay.cpp
@@ -257,4 +257,4 @@ int main (void)
     
     rc = zmq_ctx_destroy(context2);
     assert (rc == 0);
-}
\ No newline at end of file
+}
